src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -177,21 +199,21 @@ public void rename() {
             // Cannot rename remote links
             return;
         }
-        Optional<Path> fileDir = databaseContext.getFirstExistingFileDir(Globals.prefs.getFileDirectoryPreferences());
+        Optional<Path> fileDir = databaseContext.getFirstExistingFileDir(fileDirectoryPreferences);
         if (!fileDir.isPresent()) {
             dialogService.showErrorDialogAndWait(
                     Localization.lang("Rename file"),
                     Localization.lang("File directory is not set or does not exist!"));
             return;
         }
 
-        Optional<Path> file = linkedFile.findIn(databaseContext, Globals.prefs.getFileDirectoryPreferences());
+        Optional<Path> file = linkedFile.findIn(databaseContext, fileDirectoryPreferences);
         if ((file.isPresent()) && Files.exists(file.get())) {
             RenamePdfCleanup pdfCleanup = new RenamePdfCleanup(false,
                     databaseContext,
-                    Globals.prefs.getCleanupPreferences(Globals.journalAbbreviationLoader).getFileNamePattern(),
-                    Globals.prefs.getLayoutFormatterPreferences(Globals.journalAbbreviationLoader),
-                    Globals.prefs.getFileDirectoryPreferences(), linkedFile);
+                    cleanupPreferences.getFileNamePattern(),
+                    layoutFormatterPreferences,
```

Comment:
Actually layoutFormatterPrefernces can be removed from all cleanup ops (did this the maintable beta), but out of scope for this pr